packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples.2/BasicCompactInteractiveList.1.doc.mdx

```diff
@@ -0,0 +1,121 @@
+import { Notice } from '@cerner/terra-docs';
+import BasicCompactInteractiveList from './BasicCompactInteractiveList?dev-site-examples';
+import ResponsiveColumns from './ResponsiveColumns?dev-site-examples';
+
+# Basic Compact Interactive List
+
+### Description
+
+* The **[Compact Interactive List](/components/cerner-terra-framework-docs/compact-interactive-list/about)** component consists of unique rows. 
+  Each row has a border, which visually separates it from the other rows in the list.
+* The rows are visually stractured in semantic columns, which do not have visual separators, but provide data alighnment and allow [**keyboard navigation**](/components/cerner-terra-framework-docs/compact-interactive-list/about#keyboard-interactions). 
+  The **[columns](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-properties-table) property** provides the rules for semantic columns structure. 
+* The **[cells](/components/cerner-terra-framework-docs/compact-interactive-list/about#row) property** provides the unique row content. 
```

Comment:
Does it necessarily have to be unique? By design, most columns will contain similar content in each row.
```suggestion
* The **[cells](/components/cerner-terra-framework-docs/compact-interactive-list/about#row) property** provides the row content. 
```